### ISSUE TITLE
Invalid return type in shipping price calculation

### ISSUE THREAD

**Issue by swimmer-zone** [magmodules/magento2-channable#144]
Orders aren't added to the `sales_order` table, because of the following error:

```
Return value of Magmodules\Channable\Service\Order\Shipping\CalculatePrice::execute() must be of the type float, string returned in /vendor/magmodules/magento2-channable/Service/Order/Shipping/CalculatePrice.php:67
```

Proposed solution:
Change line 67 from `return $shippingPriceCal;` to `return (float) $shippingPriceCal;`

**Comment by swimmer-zone** [comment]
I see that there's already a PR pending for this issue: https://github.com/magmodules/magento2-channable/pull/143/commits/c857c8e1d18aa2f08f0831d27e03c030939486f5, sorry for this one :)

**Comment by Marvin-Magmodules** [comment]
HI @swimmer-zone,

Thank you for opening this issue, we are happy to share that we've just released the new v[1.6.2](https://github.com/magmodules/magento2-channable/releases/tag/1.6.2) version where we covered this issue. Thank you for the detailed issue report.

We are closing this issue now but please feel free to reopen the issue if this still occurs.
